DataDog/chaos-controller
# Implement a dry-run mode to injectors

### What does this PR do?

It implements a dry-run mode in injectors, allowing a disruption to be faked.

### Motivation

Ease multiple testing cases (will I target the right pods or nodes? will it drop the right packets? etc.).

### Testing Guidelines

- apply any disruption with the `dryRun: true` field
- look at created chaos pods logs: it should mention the dry-run mode
- manually check that the disruption is not injected (no qdiscs created for network disruption for instance)
- delete the disruption by still looking at the logs
